Sources/Features/Attributes/DefaultAttributes.swift

```diff
@@ -203,7 +208,7 @@ struct LibInfo: AttributesSource {
     var immutable: [String: Any?] {
         return ["guid": LibInfo.guid(store: UserDefaultsStore.self).uuidString,
                 "lang.name": LibInfo.applicationLangName,
-                "lang.version": BacktraceVersionNumber]
+                "lang.version": Bundle.main.releaseVersionNumber]
```

Comment:
did you test this for MacOS and iOS that it yields the right version number? Do/can we unit test this?

Reply:
On second thought, I think this populated incorrectly. See for Android: https://github.com/backtrace-labs/backtrace-android/blob/2f57da44a071d3f382b6a911db995d8d09c0e19f/backtrace-library/src/main/java/backtraceio/library/models/BacktraceData.java#L180 and Unity https://github.com/backtrace-labs/backtrace-unity/blob/f42386b496b741504b6223190bef790419e63874/Runtime/Model/BacktraceData.cs#L144

Although it's a bit of a mess, I think this property is meant to be system version. Can we add the macOS/iOS version here instead? 

Can you add this `Bundle.main.releaseVersionNumber` to `backtrace.version` instead? This is the field we use for the library's version (NOT the version of the app using our library)

Reply:
backtrace.version:Do we want Backtrace version and how are we extracting it .

Reply:
1) This version should never have been set in `lang.version`. This should be populate by iOS or macOS version.

So we want:
- `agent` is `backtrace-macOS`, `backtrace-tvOS` or `backtrace-iOS` (basically `backtrace-` + getSysname)
- `agent.version` is version of our library (see further below)
- `lang.name` is macOS, tvOS or iOS (same as `uname.sysname`) 
- `lang.version` is the macOs/tvOS/iOS version

Why it's not resolving: it seems to be a cocoa feature:
https://github.com/backtrace-labs/backtrace-cocoa/blob/6cb3b4b1538369320d750d7fd52776e614828579/Backtrace-iOS/Backtrace.h
https://stackoverflow.com/questions/31418303/get-cocoa-touch-framework-project-version-string-in-swift

You would have to find a way to resolve the library version for both cocoa and SwiftPM, I think this will do that:
https://stackoverflow.com/questions/24538215/get-version-number-of-ios-universal-framework-in-client